packages/atdgen-runtime/atdgen-runtime.2.0.0/opam

```diff
@@ -18,9 +18,9 @@ build: [
 
 depends: [
   "ocaml" {>= "4.02.3"}
-  "jbuilder" {>= "1.0+beta7"}
+  "jbuilder" {>= "1.0+beta17"}
   "biniou" {>= "1.0.6"}
-  "yojson" {>= "1.2.1"}
+  "yojson" {>= "1.2.1" & < "2.0.0"}
```

Comment:
```suggestion
  "yojson" {>= "1.3.1" & < "2.0.0"}
```
Due to [Unbound module Yojson.Safe.Util)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/3c9a30ea33444c325db73bbd0f999c4dac03d37a/variant/opam-2.1,compilers,4.03,atdgen-runtime.2.0.0,lower-bounds)